### PR TITLE
add spark_quick_start.py

### PR DESCRIPTION
**Pull request by bigsheeper**

### CODE REVIEW

File: scripts/spark_quick_start.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+pos1=(-73.991504, 40.770759)
+pos2=(-73.945155, 40.783434)
+limit_num=200
+
+# draw map
+
+pickup_sql = "select st_point(pickup_longitude, pickup_latitude) as point from nyc_taxi where " \
+f"(pickup_longitude between {pos1[0]} and {pos2[0]}) and (pickup_latitude between {pos1[1]} and {pos2[1]}) limit {limit_num}"
+pickup_df = spark.sql(pickup_sql)
+# 根据查询结果绘制点图图层。点大小为10，点颜色为#2DEF4A，点不透明度为1.0。
```
Comment (Emma-Song) [inline scripts/spark_quick_start.py:91]: should add space between number/English word and Chinese word. E.g., 点大小为 10
The same issue also occurs at other places in this document

Comment (bigsheeper) [reply scripts/spark_quick_start.py:91]: got it

File: scripts/spark_quick_start.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+vega = vega_pointmap(1024, 384, bounding_box=[pos1[0], pos1[1], pos2[0], pos2[1]], point_size=10, point_color="#2DEF4A", opacity=1, coordinate_system="EPSG:4326")
+res = pointmap(vega, pickup_df)
+save_png(res, '/tmp/arctern_pointmap.png')
+
+
+# 在指定地理区域（经度范围：-73.991504至-73.945155；纬度范围：40.770759至40.783434）中随机选取`200`个坐标点，并将fare_amount作为颜色权重、total_amount作为大小权重。
```
Comment (Emma-Song) [inline scripts/spark_quick_start.py:97]: `200` is a markdown format, should not be used here. Just 200 is ok.
The same issue also occurs at other places in this documents.

**Comment by bigsheeper** [comment]
> 1. add space between number/English word and Chinses word
> 2. change `200` to 200 and alike

fixed

**Pull request merged**